workspace_tools/targets.py

```diff
@@ -1389,17 +1389,17 @@ def __init__(self):
 class ARM_MPS2_Target(Target):
     def __init__(self):
         Target.__init__(self)
-        self.OUTPUT_EXT = 'axf'
-
-    def init_hooks(self, hook, toolchain_name):
-        hook.hook_add_binary("replace", self.output_axf)
-
-    @staticmethod
-    def output_axf(t_self, resources, elf, bin):
-        shutil.copy(elf, bin)
-        t_self.debug("Passing ELF file %s" % bin)
-
-
+#        self.OUTPUT_EXT = 'axf'
```

Comment:
it should have been removed completely, it was to bring it in line with other platforms that use .bin files. mps2 originally used .axf